lib/index.js

```diff
@@ -36,40 +36,44 @@ export default function runContentfulExport (usageParams) {
 
         log.info('Downloading ' + response.assets.length + ' assets')
 
-        var arrayOfDownloadPromises = []
-        response.assets.forEach((asset) => {
+        return Promise.map(response.assets, (asset) => {
           if (asset.fields.file) {
-            Object.keys(asset.fields.file).forEach((key) => {
-              arrayOfDownloadPromises.push(
-                downloadAsset(asset.fields.file[key].url, opts.exportDir).then(
-                  (downLoadedFile) => {
+            const locales = Object.keys(asset.fields.file)
+            return locales.reduce((chain, locale) => {
+              const url = asset.fields.file[locale].url || asset.fields.file[locale].upload
+              return chain.then(
```

Comment:
I want to avoid to have more concurrent running downloads than 10 since it is the AWS limit AFAIK.